api/controllers/blobs/get-blob.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  friendlyName: "Retrieve a blob",
+  description: "Endpoint to get blob info using its ID",
+  inputs: {
+    id: {
+      description: "The ID of the blob we're retrieving",
+      type: "string",
+      required: true,
+    },
+  },
+
+  exits: {
+    success: {
+      outputDescription: "Blob and associated fields",
+      outputType: "ref",
+    },
+    forbidden: { responseType: "forbidden" },
+    notFound: {
+      description: 'The requested ID does not exist',
+      responseType: 'notFound'
+    },
+  },
+
+  fn: async ({ id }) => {
+    sails.log(`Retrieving blob with ID ${id}`);
+
+    // Initialize response body
+    const responseBody = {};
+
+    // Check if blob with { id } exists
+    if (!(await Blobs.findOne({ id }))) throw 'notFound';
```

Comment:
Sweet. Might be able to wrap this up with line 34, but good for now.

Reply:
👍 I'll add it as a TODO in my next PR